### PR TITLE
Added simple benchmark for tuple_access_strategy

### PR DESCRIPTION
**Pull request by mbutrovich**
The benchmark can be invoked individually like a Google Test, or all benchmarks can be run with "make runbenchmark"

Not sure it's a valuable benchmark, but I wanted a microbenchmark in order to test the framework and demonstrate how to write future microbenchmarks.

I also formatted the benchmark and test folders.

### CODE REVIEW

File: test/include/util/container_test_util.h [hunk 1]
```diff
@@ -3,11 +3,10 @@
...
-                          const std::bitset<num_elements> &reference) {
+template <typename Bitmap, uint32_t num_elements>
+void CheckReferenceBitmap(const Bitmap &tested, const std::bitset<num_elements> &reference) {
   for (uint32_t i = 0; i < num_elements; ++i) {
     EXPECT_EQ(reference[i], tested[i]);
   }
 }
-}
\ No newline at end of file
+}  // namespace terrier
```
Comment (tli2) [inline test/include/util/container_test_util.h]: nit: new line at end of file.

File: test/include/common/test_util.h [hunk 2]
```diff
@@ -104,9 +95,9 @@ void CheckNotInBounds(A *val, B *lower, C *upper) {
...
-template<typename A>
+template <typename A>
 A *IncrementByBytes(A *ptr, uint64_t bytes) {
   return reinterpret_cast<A *>(reinterpret_cast<byte *>(ptr) + bytes);
 }
-}
-}
\ No newline at end of file
+}  // namespace testutil
+}  // namespace terrier
```
Comment (tli2) [inline test/include/common/test_util.h]: nit: new line at end of file.

File: test/include/common/harness.h [hunk 3]
```diff
@@ -6,13 +6,10 @@
```
Comment (tli2) [inline test/include/common/harness.h]: nit: This looks like a lot of format changes for some reason, did we not run style formatter on test, or did we somehow change the style checks?

Comment (mbutrovich) [reply test/include/common/harness.h]: "I also formatted the benchmark and test folders."

**Pull request merged**